src/Admin/AdminExtensionInterface.php

```diff
@@ -66,7 +66,7 @@ public function configureTabMenu(
     );
 
     /**
-     * @param mixed $object
+     * @param object $object
```

Comment:
You can remove the doc and use type hints here

Reply:
Maybe we should do the cleanup in another PR.

Reply:
I really prefer to do the cleanup in other PR.